src/muuntaja/config/common.js

```diff
@@ -16,3 +19,32 @@ export const validationOff = {
   noAdditionalProperties: false, // Do not allow additional properties in fields
   strict: false // If true, set all validationOptions to true
 };
+
+
+// if environment is test
+//   - returns the year that is given as options property 'year'
+// otherwise returns the current year in format YYYY as integer
+export function getYear(options) {
```

Comment:
Options could be simplified to boolean value isTestRun and false as default value.
If format is wanted option in this it can be required variable.

```js
export function getYear(format = 'YYYY', isTestRun = false) {
```

OR

Format and test run boolean can be "mixed"
```js
export function getYear(format = 'YYYY') {
...
if (format === 'test') {
 return '2000';
}
```

Reply:
Function was modified with the "mixed" approach.